src/alfasim_sdk/testing/fixtures.py

```diff
@@ -66,7 +66,10 @@ def load_base(self, *parts: Union[str, Path]) -> None:
         self.check_not_run()
         self.check_base_case_is_not_loaded()
 
-        self._case = convert_alfacase_to_description(Path(*parts))
+        if len(parts) == 1 and isinstance(parts[0], CaseDescription):
```

Comment:
The docstring `Define the base case from an alfacase file.` for this method may need to be updated since it may now receive either a `CaseDescription` or an `alfacase`

Reply:
Moreover, this ideally should be

```python
def load_base(self, *parts: Union[str, Path], case_description: Optional[CaseDescription] = None) -> None:
```

and then

```python
if case_description is not None:
    self._case = case_description
else:
    self._case = convert_alfacase_to_description(Path(*parts))
```

The current implementation does not make it explicit that `load_base` can receive a `CaseDescription`.

Reply:
Nice suggestion. 

- [ ] Update docstring
- [ ] Add new parameter `case_description`

Reply:
did you commit the code? It seems like the original one